bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.ceiling4.json

```diff
@@ -56,7 +56,7 @@
 				"friendlyName": "Ambient Brightness",
 				"channel": "ambientBrightness",
 				"channelType": "ambientBrightness",
```

Comment:
The channelType needs to be removed in that case otherwise the type in the json file (processing logic) has a mismatch with the type in the commonChannels.xml 
By removing the reference the channel will be fully driven by the json file.